src/util/http.ts

```diff
@@ -58,3 +59,11 @@ export const createErrorRes = ({
     }),
   };
 };
+
+export const decodeToken = ((token : any) => {
```

Comment:
```suggestion
export const decodeToken = ((token : any)  : null | JwtPayload | string => {
```
리턴 타입을 명시하는 것이, 타입스크립트 빌드시에 타입추론하기가 더욱 더 용이해집니다.

Reply:
JwtPayLoad는 제가 따로 DTO 작성해서 만들어야 되는건가요 아니면 JWT 에서 지원해주는 타입인건가요?

Reply:
JWT에서 기본적으로 지원하는 타입이에용~